wsagent/che-core-git-impl-jgit/src/main/java/org/eclipse/che/git/impl/jgit/JGitConnection.java

```diff
@@ -1696,4 +1700,42 @@ private String cleanRemoteName(String branchName) throws GitException {
         }
         return returnName;
     }
+
+    /**
+     * Method for generate exception message. The default logic return message from the error.
+     * It also check if the type of the message is for SSL or in case that the error
+     * start with "file name to long" then it raise the relevant message
+     *
+     * @param error
+     *        throwable error
+     * @return exception message
+     */
+    private String generateExceptionMessage(Throwable error) {
+        String message = error.getMessage();
+        final String FILE_NAME_TOO_LONG_ERROR_PREFIX = "File name too long";
+        //if e caused by an SSLHandshakeException - replace thrown message with a hardcoded message
```

Comment:
I think this comment should be placed above `if (error.getCause() instanceof SSLHandshakeException) {` line